src/imex.rs

```diff
@@ -805,19 +805,12 @@ mod tests {
         assert!(msg.contains("-----END PGP MESSAGE-----\n"));
     }
 
-    fn ac_setup_msg_cb(ctx: &Context, evt: Event) -> libc::uintptr_t {
-        match evt {
-            Event::GetString {
-                id: StockMessage::AcSetupMsgBody,
-                ..
-            } => unsafe { "hello\r\nthere".strdup() as usize },
-            _ => logging_cb(ctx, evt),
-        }
-    }
-
     #[test]
     fn otest_render_setup_file_newline_replace() {
```

Comment:
typo?

Reply:
yip, thanks.

Reply:
well, but this way you can sing the function name in the melody of janis joplin's "mercedes benz" :)